hubble-playground/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_MAPBOX_ACCESS_TOKEN='pk.eyJ1IjoicmF5bW9uZGRhc2h3dSIsImEiOiJja2M2bTAzeDMwMW1tMnNtZjEyYXg4NDJ2In0.wtXUl5ZNiG8bssZMriO_cw'
```

Comment:
Oh, you'll want to remove this and refresh your access token. I haven't used `.env` files before, is this a node feature? Might want to add `.env` to `.gitignore` to prevent future issues.